src/directives/automation/editor/dpEditorTemplate.js

```diff
@@ -22,9 +22,31 @@
       scope.templates = [];
       scope.isLoading = true;
 
+      const TEMPLATES = {
+        EMPTY: 0,
+        REVALIDATION_SUBSCRIBERS: 1,
+        BIRTHDAY_GREETING: 2,
+        WELCOME_NEW_CONTACTS: 3,
+        ABANDONED_CART: 4
+      };
+
+      const templateCardOrder = [
+        TEMPLATES.EMPTY,
+        TEMPLATES.WELCOME_NEW_CONTACTS,
+        TEMPLATES.BIRTHDAY_GREETING,
+        TEMPLATES.ABANDONED_CART,
+        TEMPLATES.REVALIDATION_SUBSCRIBERS
+      ];
+
+      function sortTemplateCards(templatesCardsUnsorted){
+        return templateCardOrder.reduce((templates, template) => {
+          return templates.concat(templatesCardsUnsorted[template])
```

Comment:
No seria mas claro `templateId`?
Y, por otro lado, dado que `templatesCardsUnsorted` viene de [aca](https://github.com/FromDoppler/doppler-automation-editor/blob/e7da7bd0e7cdafd32f00d91e9f5c71b287d6b2bd/src/services/automation/taskService.js#L68), es seguro que se accederá al item correcto de esta manera: `templatesCardsUnsorted[template]`? no va a depender del orden que venga del backend? (es realmente una duda, no estoy seguro, me marea un poco el reducer)
```suggestion
        return templateCardOrder.reduce((templates, templateId) => {
          return templates.concat(templatesCardsUnsorted[templateId])
```

Reply:
si por eso, reforzamos la busqueda con el find, en el commit de refactorizacion